package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
and this too? if they are not doing anything, you should remove them.

Reply:
Those files were created after running 'npm install' on my system.